charts/seed-monitoring/charts/grafana/values.yaml

```diff
@@ -19,4 +19,26 @@ extensions:
   dashboards: ""
 
 konnectivityTunnel:
-  enabled: false
\ No newline at end of file
+  enabled: false
+
+exposedComponents:
+- dashboardName: "Logging Kube Apiserver"
+  jobName: "kube-apiserver"
+  podPrefix: "kube-apiserver"
+  fileName: "logging-kube-apiserver.json"
+- dashboardName: "Logging Kube Controller Manager"
+  jobName: "kube-controller-manager"
+  podPrefix: "kube-controller-manager"
+  fileName: "logging-kube-controller-manager.json"
+- dashboardName: "Logging Kube Scheduler"
+  jobName: "kube-scheduler"
+  podPrefix: "kube-scheduler"
+  fileName: "logging-kube-scheduler.json"
+- dashboardName: "Logging Cloud Controller Manager"
```

Comment:
Cloud controller dashboard should be added via the extensions. See the [provider-aws](https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/internal/seed-controlplane/charts/cloud-controller-manager/ccm-monitoring-dashboard.json) for an example

Reply:
Okey, then I can just add a logging panels CCM dashboard of all providers?

Reply:
Done :)